src/ngx_http_lua_socket_tcp.c

```diff
@@ -93,6 +93,8 @@ static int ngx_http_lua_socket_cleanup_compiled_pattern(lua_State *L);
 static int ngx_http_lua_req_socket(lua_State *L);
 static void ngx_http_lua_req_socket_rev_handler(ngx_http_request_t *r);
 static int ngx_http_lua_socket_tcp_getreusedtimes(lua_State *L);
+static int ngx_http_lua_socket_tcp_binddata(lua_State *L);
```

Comment:
I'm afraid the word `bind` can be ambiguous in the socket world. Basically `bind` already means binding to a local address. Maybe we can use the word `tag` or `attach` instead?